engine/derivative.cc

```diff
@@ -639,6 +640,31 @@ namespace MathDAG
        throw error("frac is not differentiable");
   }
   
+  template <>
+  NodePtr SystemOfEquations::derivative<>
+  (const OperationDAG<OperationType::percent>& expr)
+  {
+    if (expr.arguments[0].empty())
+      return zero;
+    else
+       throw error("percent is not differentiable");
```

Comment:
% is just divide by 100. So the derivative will be 0.01 x.

Reply:
Actually, mea culpa. % will be multiple by 100, so the derivative should be 100x.

Reply:
I have been confused too. I guess when the % sign appears next to the number, the number must have been multiplied by 100. That helps me to know when to multiply instead divide by 100.